hack/dev-patch-config-gke.sh

```diff
@@ -25,6 +25,18 @@ readonly K8S_CLUSTER_REGION
 readonly K8S_CLUSTER_ZONE
 readonly SET_CLUSTER_CIDR=${SET_CLUSTER_CIDR:-true}
 
+function validate_arguments() {
+  if [ -n "${K8S_CLUSTER_REGION}" -a -n "${K8S_CLUSTER_ZONE}" ]; then
+      echo "Must set one of K8S_CLUSTER_REGION or K8S_CLUSTER_ZONE. Both are set."
+      exit 1
+  fi
+
+  if [ -z "${K8S_CLUSTER_REGION}" -a -z "${K8S_CLUSTER_ZONE}" ]; then
```

Comment:
did you mean to do this twice?

Reply:
One is `-n` and one is `-z`. It is testing for one-of. The error messages are slightly different.